Objects/exceptions.c

```diff
@@ -3536,13 +3536,36 @@ _PyExc_InitTypes(PyInterpreterState *interp)
 }
 
 
+static void
+_PyExc_FiniTypes(PyInterpreterState *interp)
+{
+    if (!_Py_IsMainInterpreter(interp)) {
+        return;
+    }
+
+    for (Py_ssize_t i=Py_ARRAY_LENGTH(static_exceptions) - 1; i >= 0; i--) {
```

Comment:
```suggestion
    for (Py_ssize_t i = Py_ARRAY_LENGTH(static_exceptions) - 1; i >= 0; i--) {
```

Reply:
Well well well. I would make the line more consistent, but I decided to merge my PR since I'm working on a looong serie of changes: see https://bugs.python.org/issue46417

The CPython C code base is far from being consistent and the PEP 7 is quite relaxed in terms of coding style.